Components/Image/image.css

```diff
@@ -0,0 +1,18 @@
+.Responsive-image{
+    width:100%;
+    height: 80vh;
+}
+.Round-img1{
+    width: 20rem;
+    height: 20rem;
+    border-radius: 50%;
+}
+.Round-img2{
```

Comment:
just one suggestion, may be assigning height for the image is needed. The image would adjust accordingly. Please ignore this comment if it seems all good to you!!